tests/test_clip.py

```diff
@@ -0,0 +1,23 @@
+import geopandas
+from geopandas.testing import assert_geodataframe_equal
+import dask_geopandas
+from .test_core import geodf_points  # noqa: F401
+
+
+def test_clip(geodf_points):  # noqa: F811
+    dask_obj = dask_geopandas.from_geopandas(geodf_points, npartitions=2)
+    dask_obj.calculate_spatial_partitions()
+    mask = geodf_points.iloc[:1]
+    mask["geometry"] = mask["geometry"].buffer(2)
+    expected = dask_obj.map_partitions(lambda gdf: geopandas.clip(gdf, mask)).compute()
```

Comment:
I would calculate the expected with geopandas in this case (`expected = geopandas.clip(geodf_points, mask)`)